src/App.tsx

```diff
@@ -1,19 +1,22 @@
 import React, { useState, useEffect } from "react";
-import { Heading, Link } from "@chakra-ui/react";
+import { Heading, Link, Text } from "@chakra-ui/react";
 import { Link as RouterLink } from "react-router-dom";
 
 import { getAllData } from "./util/index";
 
-const URL = "http://localhost:8000/api/v1/";
+const URL = "http://localhost:3000/api/v1/";
 
 function App() {
-  const [message, setMessage] = useState("");
+  const [message, setMessage] = useState<string>("");
+  const [textColor, setTextColor] = useState<string>("black");
 
   useEffect(() => {
-    (async () => {
-      const myData = await getAllData(URL);
-      setMessage(myData.data);
-    })();
+    getAllData(URL)
+      .then((response) => setMessage(response.data))
+      .catch((error) => {
```

Comment:
Catching the error here will ensure that the typescript error you've been seeing doesn't show up when the API isn't working. Instead, the error message will display on the page.